src/lib/Files.svelte

```diff
@@ -105,19 +119,28 @@
         })
     }
 
-    function changeViewDirectory(fileEntry: FileTreeType): void {
-        currentPath = fileEntry.path
-        viewFiles = []
-        if (fileEntry.children) {
-            fileEntry.children.forEach((file: FileTreeType) => {
+    function addViewFile(fileEntry: FileTreeType) {
+        fileEntry?.children?.forEach((file: FileTreeType) => {
+            if (!file.name.startsWith(".thumbnails") && !file.name.endsWith(".tif")) {
                 viewFiles.push({
                     fileTree: file,
                     imageSource: convertFileSrc(file.path)
                 })
-                if (file.path.endsWith(".tif")) {
-                    createThumbnail(file.path)
-                }
-            })
+            }
+            if (file.name.endsWith(".tif")) {
+                createThumbnail(file.path)
+            }
+            if (file.children && file.path.endsWith(".thumbnails")) {
+                addViewFile(file)
+            }
```

Comment:
Tenker det hadde vært mer lettlest med en switch, eller skulle det egentlig vært at det IKKE slutter på ".thumbnails" for children? 

Uansett legg gjerne til en kommentar på hva du faktisk prøver å få til, var vanskelig å henge med.

Reply:
La til kommentarer på første og siste if-statements

Reply:
Ahh nå skjønner jeg hva som faktisk skjer🤦 
Vil nok aldri skje med koden as-is, men kanskje bytt fra "if" til "else if" bare for å unngå at to ting skjer hvis vi legger til mer i framtiden.

Reply:
done done done